src/silkaj.py

```diff
@@ -71,6 +73,11 @@ def manage_cmd(ep, c):
         difficulties(ep)
 
     elif c.subcmd == "network":
+        from commands import set_network_sort_keys
```

Comment:
May be using only `-s` or `--sort`.
That's annoying it's not well implemented on `CommanLine`.